### PR TITLE
[Merged by Bors] - chore(Algebra/GroupPower/Order): Fix "See also" comment

### PR DESCRIPTION
**Pull request by YaelDillies**
Reported by Alex [here](https://github.com/leanprover-community/mathlib4/commit/d61c95e1653dffe3f92c8927a905826929f50bce)

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by alexjbest** [comment]
LGTM

maintainer merge

File: Mathlib/Algebra/GroupPower/Order.lean [hunk 1]
```diff
@@ -126,7 +126,7 @@ theorem pow_lt_pow_left (h : x < y) (hx : 0 ≤ x) : ∀ {n : ℕ}, n ≠ 0 →
       mul_lt_mul_of_le_of_le' (pow_le_pow_left hx h.le _) h (pow_pos (hx.trans_lt h) _) hx
 #align pow_lt_pow_of_lt_left pow_lt_pow_left
 
-/-- See also `pow_left_strictMonoOn`. -/
+/-- See also `pow_left_strictMono` and `Nat.pow_left_strictMonoOn`. -/
```
Comment (alexjbest) [inline Mathlib/Algebra/GroupPower/Order.lean:129]: actually does this second lemma exist?!

Comment (ericrbg) [reply Mathlib/Algebra/GroupPower/Order.lean:129]: The names are the wrong way round, cf https://loogle.lean-lang.org/?q=%22pow_left_strictMono%22

Comment (YaelDillies) [reply Mathlib/Algebra/GroupPower/Order.lean:129]:
```suggestion
/-- See also `pow_left_strictMono` and `Nat.pow_left_strictMono`. -/
```

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**